py/specs/sys_shards.toml

```diff
@@ -0,0 +1,13 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
+statements = [
+    "copy uservisits from 's3://crate.amplab/data/tiny/uservisits/part-*' with (compression = 'gzip')",
```

Comment:
There is no need for data for a `sys.shards` benchmark if the table isn't partitioned